pkg/daemon/update.go

```diff
@@ -176,6 +178,7 @@ func (dn *Daemon) updateFiles(oldConfig, newConfig *mcfgv1.MachineConfig) error
 // it encounters.
 func (dn *Daemon) deleteStaleData(oldConfig, newConfig *mcfgv1.MachineConfig) {
 	var path string
+	glog.Info("Deleting stale data")
```

Comment:
Is it too verbose to log each file that is being deleted, or would that be useful information when debugging?

Reply:
I personally think this is nice. It's letting us know that garbage collection is occurring. If this ends up being called once or twice a minute then I think we should up the `V` level. But for now :+1:

Reply:
Ok I'll leave this for now then, but of course am happy to change the logging level/ remove logging as people see fit.